### DISCUSSION TITLE
org-cycle-separator-lines: Blank lines not shown between folded headlines

### DISCUSSION THREAD

**Discussion by bsvh** [minad/org-modern#165] [Q&A]
I have `org-cycle-separator-lines` set to `1` so that blank lines are shown between headings  when using `org-cycle` , which is my preference. When `org-modern-mode` is turned on, there are no blank lines between any headings no matter how many newlines are between them. Is there any way to configure this with `org-modern`?

**Comment by minad** [comment]
Hi, thanks for the report. Can you please investigate which one of the org-modern customization settings is the cause of the problematic interaction, e.g., org-modern-star? You can try the various customization values. Then we can see what we can do about it or at least document the incompatibility. Thanks!

**Reply by bsvh** [reply]
It looks like it happens when `org-modern-hide-stars` is set to `t`. It works fine when it is set to `nil` or `'leading`.
